README.md

```diff
@@ -39,7 +41,7 @@ django-admin --version
 ### Connect to the database
 
 Navigate to your `example/settings.py` file and replace the `DATABASES`
-setting like so:
+setting using your cluster's [connection string](https://www.mongodb.com/docs/manual/reference/connection-string/) like so:
```

Comment:
Is it okay to talk about "your cluster". In my development, I've used the standalone products. Not sure if this could give the impression this only works on a cluster.